From 118dd56b0aa4ad06097b96be0577b4636c1685e3 Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Tue, 12 Jan 2016 15:12:03 +0800
Subject: [PATCH 1403/1691] MLK-12211-1: ARM: dts: imx6/imx7: enable
 snvs-poweroff

commit 118dd56b0aa4ad06097b96be0577b4636c1685e3 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

enable snvs-poweroff driver on imx6sx(except sabreauto board),imx6ul and
imx7d all boards.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
---
 arch/arm/boot/dts/imx6sx-14x14-arm2.dts | 4 ++++
 arch/arm/boot/dts/imx6sx-19x19-arm2.dts | 4 ++++
 arch/arm/boot/dts/imx6ul.dtsi           | 7 +++++++
 arch/arm/boot/dts/imx7d.dtsi            | 7 +++++++
 4 files changed, 22 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-14x14-arm2.dts b/arch/arm/boot/dts/imx6sx-14x14-arm2.dts
index 42a903b..5eeb064 100644
--- a/arch/arm/boot/dts/imx6sx-14x14-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-14x14-arm2.dts
@@ -464,6 +464,10 @@
 	status = "okay";
 };
 
+&snvs_poweroff {
+	status = "okay";
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1_1>;
diff --git a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
index bcef325..044c078 100644
--- a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
@@ -425,6 +425,10 @@
 	status = "disabled";
 };
 
+&snvs_poweroff {
+	status = "okay";
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1_1>;
-- 
1.9.1

